mysql/resource_user_password.go

```diff
@@ -0,0 +1,98 @@
+package mysql
+
+import (
+	"fmt"
+	"github.com/satori/go.uuid"
+
+	"github.com/hashicorp/terraform/helper/encryption"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceUserPassword() *schema.Resource {
+	return &schema.Resource{
+		Create: SetUserPassword,
+		Read:   ReadUserPassword,
+		Delete: DeleteUserPassword,
+
+		Schema: map[string]*schema.Schema{
+			"user": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"host": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "localhost",
+			},
+
+			"pgp_key": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
```

Comment:
Should this be marked sensitive?

Reply:
Nah, it's just a public key. It can be `keybase:joestump` for instance.